pay-queue/src/pay_queue/external/pubsub.py

```diff
@@ -78,11 +78,11 @@ def init_app(self, app: Flask):
         self.gcp_auth_key = app.config.get('GCP_AUTH_KEY')
         if self.gcp_auth_key:
             try:
-                audience = app.config.get(
+                audience = current_app.config.get(
```

Comment:
What's the difference here between current_app and app?

Reply:
I removed this change. filer used current_app, but it changed back when I check today.